src/article_metrics/management/commands/import_metrics.py

```diff
@@ -61,13 +84,13 @@ def handle(self, *args, **options):
                     time.sleep(1)
 
                 except BaseException as err:
-                    LOG.exception("unhandled error in source %s: %s", source, err)
+                    DEBUG_LOG.exception("unhandled error in source %s: %s", source, err)
                     continue
 
             end_time = time.time()
 
             elapsed_seconds = end_time - start_time
-            LOG.info("time elapsed: %s minutes" % elapsed_seconds * 60)
+            DEBUG_LOG.info("time elapsed: %s minutes" % elapsed_seconds * 60)
```

Comment:
this is interesting. search for this in Loggly

but also delete, as we're handling it in a better way now